config/chalupa-vr.json

```diff
@@ -52,16 +52,19 @@
       },
       {
          "SlaveAddress":"0x13",
+         "PmbusAddress":"0x43",
```

Comment:
just thinking out loud...Do we even need "PmbusAddress" field? As it looks like SlaveAddress + PmbusTransOffset, where PmbusTransOffset = 0x30 would work without needing so many changes. Thoughts?